### ISSUE TITLE
Privacy badger breaks hCaptcha

### ISSUE THREAD

**Issue by 161134050** [EFForg/privacybadger#2574]
Cloudflare is now using hCaptcha as their Captcha service, and Privacy badger seems to be blocking their domain

#### What is your browser and browser version?
Latest Chrome 80. something

#### What is broken and where?
https://www.unknowncheats.me/forum/other-games/381331-raceroom-racing-experience.html

#### What is the "culprit" domain?
hCaptcha.com

#### What is your debug output for this domain?
The captcha requiest is blocked by client

**Comment by 161134050** [comment]
I've set up a site and put it behind cloudflare with a rule to Captcha all users for debugging purposes:

**Comment by ghostwords** [comment]
Hello and thanks for the report! Does moving the slider for `hcaptcha.com` domains to "yellow" and reloading the page work around the problem?

**Comment by 161134050** [comment]
> Hello and thanks for the report! Does moving the slider for `hcaptcha.com` domains to "yellow" and reloading the page work around the problem?

Yes, however both of these need to be yellow, not just the top one 
![image](https://user-images.githubusercontent.com/39843740/78296606-9bd59f00-7560-11ea-8c06-046f680dc57b.png)
